lib/authc/AssertionAuthenticationResult.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var nJwt = require('njwt');
+var Account = require('../resource/Account');
+
+function AssertionAuthenticationResult(dataStore, data) {
```

Comment:
It's just a authentication result. I don't understand how that would be connected to a resource. I think it's also strange that AuthenticationResult lies where it do, since that itself isn't a resource.